modules/programs/neomutt-accounts.nix

```
@@ -21,6 +21,27 @@ let
   };
 
 in {
+  options.notmuch.neomutt = {
+    enable = mkEnableOption "Notmuch support in NeoMutt" // {
+      default = true;
+    };
+
+    virtualMailboxes = mkOption {
+      type =
+        types.listOf (types.submodule (import ./notmuch-virtual-mailbox.nix));
```

```suggestion
        types.listOf (types.submodule ./notmuch-virtual-mailbox.nix);
```

That way options in the manual will point to that file.